gamedata/movedefs.lua

```diff
@@ -203,8 +203,8 @@ local moveDatas = {
 		badslope = 22,
 		badwaterslope = 255,
 		crushstrength = 25,
-		footprintx = 3,
-		footprintz = 3,
+		footprintx = 4,
+		footprintz = 4,
```

Comment:
Shouldn't this be called HOVER4 now? It seemed the name was intended to match the foorptinr size,